src/breakpoints/body.tsx

```diff
@@ -88,6 +88,13 @@ const BreakpointCellComponent = ({
   breakpoints: IDebugger.IBreakpoint[];
   model: BreakpointsModel;
 }) => {
+  const removeFirstCharIfSlash = (breakpoint: IDebugger.IBreakpoint) => {
+    breakpoint.source.path =
+      breakpoint.source.path[0] === '/'
```

Comment:
It would be interesting to see how this behaves with Windows path, since they don't start with a leading `/`.

Reply:
And I guess we still we want to keep the leading `/` and not remove it.

Reply:
![Peek 2020-04-17 13-10](https://user-images.githubusercontent.com/45068804/79563501-3b407900-80ad-11ea-9f5e-94b46acff620.gif)
![Peek 2020-04-17 12-17](https://user-images.githubusercontent.com/45068804/79563504-3da2d300-80ad-11ea-9478-4ceec9d05a83.gif)

Third solution. Now path is not changed.